hugo/layouts/engine/single.html

```diff
@@ -7,7 +7,7 @@
             <br><br>
             <h1 class="title text-left p-0 mt-5 mb-4">Turn your code into actionable insights with source{d} Engine</h1>
             <a href="https://go.sourced.tech/engine" class="btn btn-purple-light mr-4 mb-3 mb-sm-5">Request A Demo</a>
-            <a href="https://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light mb-3 mb-sm-5">Install source{d} Engine</a>
+            <a href="https://go.sourced.tech/engine-download" class="btn btn-highlight-light mb-3 mb-sm-5">Install source{d} Engine</a>
```

Comment:
We can cut the source{d} from the button caption (`Install Engine` is fine), as in this particular case it makes the CTA disproportionately long here and you have a ton of "source{d}" and a few "Engine" around the navbar/hero/header so it's not something you'd miss.

Reply:
Thanks @marnovo ! Done.